pkg/model/keycloak_deployment.go

```diff
@@ -152,6 +152,10 @@ func getKeycloakEnv(cr *v1alpha1.Keycloak, dbSecret *v1.Secret) []v1.EnvVar {
 			Name:  "PROXY_ADDRESS_FORWARDING",
 			Value: "true",
 		},
+		{
+			Name:  "KEYCLOAK_STATISTICS",
```

Comment:
Is this variable documented somewhere? I suppose it's a var used by KC, not by the operator.

Reply:
Here's the manual for this: https://github.com/keycloak/keycloak-containers/tree/master/server#enable-some-metrics

In the Operator, we need the metrics for Grafana.